### PR TITLE
feat(unitctl): update package versions, improve request body handling

### PR DESCRIPTION
**Pull request by Pavlusha311245**
- Bump version to 1.35.6 in Cargo.toml and openapi-config.json
- Introduce `KnownSize::into_full_body()` for accurate Content-Length calculation
- Optimize process filtering in `find_unitd_processes()`
- Update dependencies to latest compatible versions
- Add error handling for I/O operations in request body materialization
- Improve test coverage for new features and edge cases

### Checklist
- [x] I have read [CONTRIBUTING.md](/CONTRIBUTING.md)
- [x] If applicable, I have added tests
- [x] If applicable, I have updated documentation

### CODE REVIEW

File: tools/unitctl/unit-client-rs/src/unitd_docker.rs [hunk 1]
```diff
@@ -142,7 +141,7 @@ impl Serialize for UnitdContainer {
 impl UnitdContainer {
     pub async fn find_unitd_containers() -> Vec<UnitdContainer> {
         if let Ok(docker) = Docker::connect_with_local_defaults() {
-            match docker.list_containers::<String>(None).await {
+            match docker.list_containers(None).await {
```
[the remark this answers is not in the document]
Comment (Pavlusha311245) [reply tools/unitctl/unit-client-rs/src/unitd_docker.rs:144]: Closed because Gemini does not account for the package versioning structure used in the project and incorrectly attempts to roll back changes

File: tools/unitctl/unit-client-rs/src/unitd_docker.rs [hunk 2]
```diff
@@ -324,7 +320,7 @@ pub async fn deploy_new_container(
                 ]);
             }
             container_conf.host_config = Some(host_conf);
-            match docker.create_container::<String, String>(None, container_conf).await {
+            match docker.create_container(None, container_conf).await {
```
[the remark this answers is not in the document]
Comment (Pavlusha311245) [reply tools/unitctl/unit-client-rs/src/unitd_docker.rs:323]: Closed because Gemini does not account for the package versioning structure used in the project and incorrectly attempts to roll back changes

File: tools/unitctl/unit-client-rs/src/unitd_docker.rs [hunk 3]
```diff
@@ -364,7 +360,7 @@ pub async fn deploy_new_container(
                     match docker
                         .start_container(
                             info[0].names.clone().unwrap()[0].strip_prefix(MAIN_SEPARATOR).unwrap(),
-                            None::<StartContainerOptions<String>>,
+                            None::<StartContainerOptions>,
```
[the remark this answers is not in the document]
Comment (Pavlusha311245) [reply tools/unitctl/unit-client-rs/src/unitd_docker.rs:363]: Closed because Gemini does not account for the package versioning structure used in the project and incorrectly attempts to roll back changes
